rule/repository_memory.go

```diff
@@ -133,5 +142,16 @@ func (m *RepositoryMemory) Match(ctx context.Context, method string, u *url.URL,
 		return nil, errors.WithStack(helper.ErrMatchesMoreThanOneRule)
 	}
 
-	return &rules[0], nil
+	return rules[0], nil
+}
+
+func (m *RepositoryMemory) ReadyChecker(r *http.Request) error {
```

Comment:
This simple function is so much nicer than that random event pipeline, with actually zero meaning in the end :sweat_smile: but that pre-dated the `ory/x/healthx` package, so the author is excused :wink:

Reply:
Agreed!